mapcompose/src/main/java/ovh/plrapps/mapcompose/ui/state/MapState.kt

```diff
@@ -56,6 +58,7 @@ class MapState(
     internal val tileSize by mutableStateOf(tileSize)
     internal var stateChangeListener: (MapState.() -> Unit)? = null
     internal var touchDownCb: (() -> Unit)? = null
+    internal val mapBackground by mutableStateOf(mapBackground)
```

Comment:
I believe this is where the default value should be:
`internal val mapBackground by mutableStateOf(Color.White)`

Reply:
Fixed